### ISSUE TITLE
[Question] Does wasm support work only via UNO? 

### ISSUE THREAD

**Issue by dr-vij** [mono/SkiaSharp#1819]
I cant find native lib for skiasharp in main solution, just android ios macos and etc. I am trying to include libSkiaSharp.a, but I cant create SKCanvases in wasm and webgl. Is it real at all?

**Comment by Gillibald** [comment]
The native bits can be used via https://www.nuget.org/packages/SkiaSharp.NativeAssets.WebAssembly/2.88.0-preview.145
You still need https://www.nuget.org/packages/Uno.Wasm.Bootstrap/ to set up Emscripten and the Mono runtime etc.
The last step would be creating a canvas and use that as a surface to render on. In the end, you just have Skia in the browser. The glue code is up to you.

**Comment by dr-vij** [comment]
Thank you for the answer! I am trying to connect SkiaSharp to Unity 3d WebGL wasm, it works on standalone and my only problem is webassembly. I didn't try to use uno wasm bootrsrap yet, thank you so much for this direction. I will try to figure out how to use it

**Comment by mattleibow** [comment]
I am also in the process of adding WASM support for Blazor apps: https://github.com/mono/SkiaSharp/pull/1811

Are you trying to get a wasm build without any uno/blazor/framework?

**Comment by dr-vij** [comment]
Yes, I have some problems with understanding of replacement of dynamic linking cause unity wasm does not support it. But unfortunately I have no success with it yet. I am doing this out of duty, I think skia has fantastic potential for texture processing in gamedev.

**Comment by dr-vij** [comment]
they do!) https://docs.unity3d.com/Manual/NativePlugins.html
they use v1.37.27 Emscripten for Unity 2018-2020, and they updated to 2.x version in 2021 unity as I know
